doc/source/whatsnew/v1.1.0.rst

```diff
@@ -758,6 +758,7 @@ ExtensionArray
 - Fixed bug where :meth:`Series.value_counts` would raise on empty input of ``Int64`` dtype (:issue:`33317`)
 - Fixed bug in :class:`Series` construction with EA dtype and index but no data or scalar data fails (:issue:`26469`)
 - Fixed bug that caused :meth:`Series.__repr__()` to crash for extension types whose elements are multidimensional arrays (:issue:`33770`).
+- Fixed bug where :meth:`Series.update` would raise a ``ValueError`` for ``ExtensionArray`` dtypes with missing values (:issue:`33980`)
```

Comment:
not sure if using the term ExtensionArray here is correct. This PR does not fix #25744 or #34020 AFAICT #34020 is a regression from 0.25.3 whereas #25744 was already failing on 0.25.3

Reply:
Hmm, I'm seeing both of these issues as being fixed now on master so I think it may close those as well (seems they all hit the same code path)?

Reply:
> Hmm, I'm seeing both of these issues as being fixed now on master

doh! quite possible that I hadn't merged master!

> so I think it may close those as well (seems they all hit the same code path)?

due to the regression, I may want to backport this. so best not to close till we have tests.

probably want to replace the tests here with tests in tests/extension instead

Reply:
> > Hmm, I'm seeing both of these issues as being fixed now on master
> 
> doh! quite possible that I hadn't merged master!
> 
> >

so I was on 1.1.0.dev0+1468.g3cebe3024 and these issues are indeed fixed on latest master